tests/TestyMcTestFace.cpp

```diff
@@ -78,12 +91,48 @@ TEST(FramePluginTests, MissingStateValue) {
          badState));;
 }
 
-/* TEST_F(FrameIsdTest, ConstructFromISD) {
+TEST_F(FrameIsdTest, Constructible) {
    UsgsAstroFramePlugin testPlugin;
    EXPECT_TRUE(testPlugin.canModelBeConstructedFromISD(
                isd,
                "USGS_ASTRO_FRAME_SENSOR_MODEL"));
-} */
+}
+
+TEST_F(FrameIsdTest, ConstructValidCamera) {
+   UsgsAstroFramePlugin testPlugin;
+   csm::Model *cameraModel;
+   EXPECT_NO_THROW(
+         cameraModel = testPlugin.constructModelFromISD(
+               isd,
+               "USGS_ASTRO_FRAME_SENSOR_MODEL",
+               NULL)
+   );
+   UsgsAstroFrameSensorModel *frameModel = dynamic_cast<UsgsAstroFrameSensorModel *>(cameraModel);
+   EXPECT_FALSE(frameModel == NULL);
```

Comment:
There's gotta be a better way to check if a pointer is NULL in gtest, but this is the best I could find.

Reply:
What about `IsNull`? https://github.com/google/googletest/blob/master/googlemock/docs/CheatSheet.md#generic-comparison

Reply:
:+1: